lib/parser.js

```diff
@@ -78,9 +81,15 @@ Parser.prototype._preprocess = function (element, context, config) {
 
     let isIncludeSrcMd = utils.getExtName(filePath) === 'md';
 
+    let actualFilePath = filePath;
+    if (!utils.fileExists(filePath)) {
+      let boilerplateFilePath = calculateBoilerplateFilePath(filePath, config);
+      actualFilePath = utils.fileExists(boilerplateFilePath) ? boilerplateFilePath : actualFilePath;
+    }
+
```

Comment:
Can we bring line 77 
```js
 this.staticIncludeSrc.push({from: context.cwf, to: filePath});
```
and change to 
```js
 this.staticIncludeSrc.push({from: context.cwf, to: actualFilePath});
```
So that we capture both boilerplate and non boilerplate files?

Reply:
Done